<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>819</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QVTKOpenGLNativeWidget" name="openGLWidget">
    <property name="geometry">
     <rect>
      <x>-2</x>
      <y>38</y>
      <width>631</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>811</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="STLButton">
       <property name="text">
        <string>STL</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sphereButton">
       <property name="text">
        <string>Sphere</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="HemisphereButton">
       <property name="text">
        <string>hemisphere</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cubeButton">
       <property name="text">
        <string>cube</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="coneButton">
       <property name="text">
        <string>cone</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pyramidButton">
       <property name="text">
        <string>pyramid</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cylinderButton">
       <property name="text">
        <string>cylinder</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="curvedCylButton">
       <property name="text">
        <string>curvCylinder</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="tubeButton">
       <property name="text">
        <string>tube</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="donutButton">
       <property name="text">
        <string>donut</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>80</y>
      <width>171</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>420</y>
      <width>48</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_2">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>420</y>
      <width>48</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_3">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>420</y>
      <width>48</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>460</y>
      <width>67</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>50</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Objects</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>510</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>ok</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>120</y>
      <width>171</width>
      <height>28</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>color&amp;opacity</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>delete</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>rotate</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>flipX</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>flipY</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>flipZ</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>translate</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>scale</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="editButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>160</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>edit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mergeButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>200</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>merge</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_3">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>240</y>
      <width>171</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>350</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="nextButton">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>460</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>next</string>
    </property>
   </widget>
   <widget class="QLabel" name="editLabel">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>300</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="saveSTLButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>510</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>saveSTL</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>819</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKOpenGLNativeWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>C:\VTK\include\vtk-9.2\QVTKOpenGLNativeWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
